File: Android/DictionaryRobot/app/src/main/java/com/example/ramsr/dictionaryrobot/model/SearchResults.java
package com.example.ramsr.dictionaryrobot.model;



public class SearchResults {



    private String keyword;
    private String definition;
    private Boolean isFetched;

    public SearchResults(String keyword, String definition, Boolean isFetched) {
        this.keyword = keyword;
        this.definition = definition;
        this.isFetched = isFetched;
    }

    public String getKeyword() {
        return keyword;
    }

    public String getDefinition() {
        return definition;
    }

    public Boolean getSearchResponse() {
        return isFetched;
    }
}

File: iOS/Dictionary Robot/Dictionary Robot/Model/SearchResults.swift
//
//  SearchResults.swift
//  Dictionary Robot
//
//  Created by Ram Sri Charan on 10/13/18.
//  Copyright © 2018 Ram Sri Charan. All rights reserved.
//

import Foundation

//------------------------------------------------------------------------------
////////////////// Model to decode and store Search Results  //////////////////
//------------------------------------------------------------------------------

struct SearchResults
{
    private var keyword : String
    private var definition : String
    private var isFetched : Bool
    
    init(Keyword : String, Definition : String, IsFetched : Bool) {
        self.keyword = Keyword
        self.definition = Definition
        self.isFetched = IsFetched
    }
    
    func getKeyword() -> String
    {
        return self.keyword
    }
    
    func getDefinition() -> String {
        return self.definition
    }
    
    func getIsFetched() -> Bool {
        return isFetched
    }
    
    
}
